File_Organiser/main.py

```diff
@@ -0,0 +1,95 @@
+import os,sys
+import json
+import shutil
+import time

```

Comment:
Please add a readme to this folder for additional libraries which need to be installed like tqdm, colorama etc.
The readme would also be helpful to guide user how to use this code. Like run main.py using python3 main.py.
What are the inputs and those stuff.
It is not clear from **Lets configure ...** then What inputs are to be put i Document and others. The readme should be like one don't need to open your code but is able to run directly following the  steps in readme.

Reply:
Yes Sir ...  I will do these changes in 2-3 days ..
Thank you for your constructive feedback!